packages/InteractiveProfilingTool-Tests.package/IPTTreeModelTestCase.class/instance/setUp.st

```diff
@@ -0,0 +1,11 @@
+as yet unclassified
+setUp
+
+	| node result |
+	self halt.
+	node := MessageTally new.
+	result := node spyEvery: MessageTally defaultPollPeriod on: [5 timesRepeat: [Project current world doOneCycle]].
```

Comment:
Shouldn't his draw on the ```stopProfiling``` method an profile until it is stopped by the user?

Reply:
Our idea was to create a just one simple *MessageTally* object that we can use in our test. The Idea behind this was that we don't need all the time a user interaction to start the tests.

Reply:
> Our idea was to create a just one simple MessageTally object that we can use in our test.

Actually, I think that’s the preferred way to test this class as you usually want to avoid accessing/testing other parts of the codebase in unit tests.